packages/wallet-link/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@helium/wallet-link",
+  "version": "4.4.0",
```

Comment:
Should this be 1.0.0?

Reply:
all the lerna package versions are linked to update together, so this would start at whatever the current version overall is

Reply:
I'm not sure. @allenan ?

Reply:
I ran `learn create` and it generated this for me 🤷

Reply:
Ah ok no problem then